### PR TITLE
EWL-4205 Adds copyright atom

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**
- [EWL-4205: copyright atom](https://issues.ama-assn.org/browse/EWL-4205)

## Description

Adds copyright atom

## To Test

- [ ] gulp serve

## Visual Regressions

Does it match SG1?

## Relevant Screenshots/GIFs

screen shot 2017-12-07 at 12 50 32 pm

## Remaining Tasks

n/a

## Additional Notes
n/a

### CODE REVIEW

File: styleguide/source/_patterns/01-atoms/copyright.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{# Conditionally checks the current year to automatically update the copyright range. #}
+{% set startYear = 1995 %}
+{% set currentYear = "now"|date("Y") %}
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/copyright.twig]: This is fabulous!

File: styleguide/source/_patterns/01-atoms/copyright.twig [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+{% if startYear == currentYear %}
+    {% set copyrightYears = startYear %}
+{% else %}
+    {% set copyrightYears = startYear ~ ' - ' ~ currentYear %}
+{% endif %}
+
+<p class="ama__copyright ama__type-smaller">
+  {{ copyright.begin }} {{ copyrightYears }} {{ copyright.end }}
+</p>
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/copyright.twig]: Line ending

File: styleguide/source/assets/scss/01-atoms/_copyright.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.ama__copyright {
+  color: $black-55;
+}
```
Comment (labbydev) [inline styleguide/source/assets/scss/01-atoms/_copyright.scss]: line ending

File: styleguide/source/_patterns/01-atoms/copyright.twig [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+{% set startYear = 1995 %}
+{% set currentYear = "now"|date("Y") %}
+
+{% if startYear == currentYear %}
+    {% set copyrightYears = startYear %}
+{% else %}
+    {% set copyrightYears = startYear ~ ' - ' ~ currentYear %}
+{% endif %}
+
+<p class="ama__copyright ama__type-smaller">
```
Comment (labbydev) [inline styleguide/source/_patterns/01-atoms/copyright.twig]: The class `ama__type-smaller` should use a double hyphen for the modifer making it `ama__type--smaller`.  Using a `<p>` tag for this is a change and gives extra margins around the copyright. This likely needs to continue being  a `<div>` to retain the proper spacing.

Comment (chelsiejohnston) [reply styleguide/source/_patterns/01-atoms/copyright.twig]: I agree that it should be `div` since we don't want to imply spacing at the atom level?

**Comment by designerneil** [comment]
Thanks guys. Ive updated the PR per your comments

**Comment by designerneil** [comment]
@chelsie i added a default timezone. but you really shouldn't be getting any errors

**Comment by chelsiejohnston** [comment]
@designerneil no more error! That did it. Merging!

**Pull request merged**